### ISSUE TITLE
Signal Intersection Aerial PDFs to ArcGIS Pro, Web Map/App

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-data-tech#2200]
Need to update the process for the signal intersection aerial PDFs to run in ArcGIS Pro with Python 3.

**Comment by jaime-mckeown** [comment]
Discovery Day:
- What was your mission?
- Was it hard?
- What did you learn?
Slide, visual, demo, screen share, etc.

**Comment by jaime-mckeown** [comment]
- All 3 map series can be added to the same project, each under a separate map layout.
- Since the buffers will need to be remade anytime a new signal is added, I added a new field to the buffer layers that concatenates the Extent, Signal ID, Primary St, and Cross St to name the PDFs using the Extent, Signal ID and Location Name of the intersection.
- Right now the script will only run when the Project is open so the Nearmap imagery will draw. I need to figure out how to connect to AGOL via the script so the Project does not need to be open. All 3 extent Map Series can be created with the script, so at least no interaction is required once it starts.

**Comment by jaime-mckeown** [comment]
The 300' PDFs keep hanging up on the Export. I need to figure out what's going on with the Export  sometime over the next Sprint.

**Comment by jaime-mckeown** [comment]
Certain pages in the Map Series absolutely refuse to export. I've tried many different setups, but still having issues. Hopefully this will be resolved during the next Sprint.

**Comment by jaime-mckeown** [comment]
- When I was talking with John about this he asked why a Web Map couldn't be used. I honestly did not think about using a Web Map, but it makes more sense then having 3,000+ PDFs sitting on the server that will need to be updated periodically.
- I created a Web Map and App with the ability to print and sent a link to Sukumar to see what he thinks instead of having all the PDFs.

**Comment by jaime-mckeown** [comment]
I reached out to Sukumar yesterday about the Web App. He prefers the App, but is waiting to hear back from Brian and the Signal Engineers for additional feedback.

**Comment by jaime-mckeown** [comment]
Send Sukumar a message today to see if he's gotten any feedback from the group.

**Comment by jaime-mckeown** [comment]
- Neither ask from AMD is possible. CTM controls the AGOL Map Templates which are very simple. Ben also let me know that the scale bar is not included as it did not work well in the map.
- To try and mimic what AMD is asking for I added 4 buffers to the map: 50', 100', 150', 300'. These layers will help the engineers to see what scale they are currently viewing and to help when printing the PDF.
- Specific Zoom Extents were set in the App: closest zoom - 50' (added in case they need to zoom closer then originally had asked), 2nd closest zoom - 100', 3rd closest zoom - 150', 4th closest zoom - 300'. The buffers can be left turned on or toggled off if they don't want them to be in the PDF map.
- The Nearmap imagery is only visible at the 5 closest zoom levels to help the map draw faster at zoomed out levels.
- I created a Wiki for the App that includes descriptions of each Tool included in the map, focusing on how best to use the Print Tool. https://app.gitbook.com/@atd-dts/s/atd-geospatial/signal-intersection-aerials-map
- Sent Sukumar a link to the Wiki and asked him to go over what I have done to see how it will work with the changes that have been implemented.
